platform/podjs/src/browserPod.ts

```diff
@@ -262,14 +308,13 @@ class IDBPolyOut implements PolyOut {
                 },
                 stat() {
                     const { uncompressedSize, directory, lastModDate } = entry;
```

Comment:
Nit: I would get rid of the destructuring here, and just use `entry.` below. The reason I assigned those local variables in the first place was to prevent the closure of the `get*()` functions to retain a reference to the `entry` object. But that's no longer a concern. Same below in the other code path.

Reply:
Done.